src/containers/title/index.js

```diff
@@ -0,0 +1,27 @@
+import React, { useState, useMemo} from "react";
+import "./index.sass";
+
+const colors = ['#f55d42', '#f5dd42', '#f5429e', '#42f569', '#03f8fc' ];
+
+const Title = () => {
+    const [color, setColor] = useState(colors[0]);
+    
+    const style = useMemo(() => {
+        return { color };
+    }, [color])
+
+    const handleHover = () => {
+        const rand =  Math.random() * (4 + 1);
```

Comment:
Math.random() * (4 + 1) you can change to Math.random() * colors.length. And when you'll add new colors into colors it will work without changing const rand =  Math.random() * (4 + 1);